astropy/stats/sigma_clipping.py

```diff
@@ -194,6 +208,18 @@ def perform_clip(_filtered_data, _kwargs):
         warnings.warn("Input data contains invalid values (NaNs or infs), "
                       "which were automatically masked.", AstropyUserWarning)
 
+    if masked is False and HAS_SCIPY is True:
+        if cenfunc is not np.ma.mean or stdfunc is not np.std:
+            warnings.warn("Using np.mean and np.std to calculate the"
+         "sigma limits, ignoring user defined functions..", AstropyUserWarning)
+
+        filtered_data = sigmaclip(data, sigma_lower, sigma_upper)
+        return filtered_data
+    elif masked is False and HAS_SCIPY is False:
+        warnings.warn("Scipy is not present - Cannot use the",
+                "scipy.stats.sigmaclip function. Falling back on ",
+                "astropy.stats.sigma_clip function.", AstropyUserWarning)
```

Comment:
you actually doesn't set ``masked`` to True when ``HAS_SCIPY`` is False. Maybe it would be also more clear if it would be a nested if/else and first tests for ``HAS_SCIPY``.